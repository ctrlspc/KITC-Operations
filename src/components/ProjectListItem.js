import React from 'react';
import { Link } from 'react-router-dom';

export default ({ id, title, projectManager, projectType }) => (
  <Link className="list-item" to={`/project/${id}`}>
    <div>
      <h3 className="list-item__title">{title}</h3>
      <p className="list-item__subtitle">{projectManager.name}</p>
    </div>
    <p className="list-item__data">{projectType}</p>
  </Link>
);