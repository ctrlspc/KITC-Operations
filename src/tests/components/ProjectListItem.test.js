import React from 'react';
import { shallow } from 'enzyme';
import ProjectListItem from '../../components/ProjectListItem';

test('should render the ProjectListItem component correctly', () => {
  const wrapper = shallow (<ProjectListItem projectManager={{name:'test'}}/>);
  expect(wrapper).toMatchSnapshot();
});