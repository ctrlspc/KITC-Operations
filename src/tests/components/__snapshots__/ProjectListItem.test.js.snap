// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the ProjectListItem component correctly 1`] = `
<Link
  className="list-item"
  replace={false}
  to="/project/undefined"
>
  <div>
    <h3
      className="list-item__title"
    />
    <p
      className="list-item__subtitle"
    />
  </div>
</Link>
`;
